src/shadowbox/server/manager_metrics.ts

```diff
@@ -15,8 +15,22 @@
 import {PrometheusClient} from '../infrastructure/prometheus_scraper';
 import {DataUsageByUser, DataUsageTimeframe} from '../model/metrics';
 
+export type TunnelTimeDimension = 'access_key' | 'country' | 'asn';
+
+interface TunnelTimeRequest {
+  hours: number;
```

Comment:
This should probably use the same duration object as the Response.

Also, it helps to use a name that represents what it is, like `window_length` or `time_window`?